spec/shortcuts_spec.rb

```diff
@@ -142,6 +142,14 @@
       m2 = NMatrix.random(2,:seed => 62)
       m3 = NMatrix.random(2,:seed => 65)
 
+      expect(m1.stype).to eq(:dense)
+      expect(m1.dtype).to eq(:float64)
+      
+      expect(m2.stype).to eq(:dense)
```

Comment:
And as a follow-up, if you think they're necessary, please justify their inclusion (reply to this comment) — don't just remove them. NMatrix contributors are not gods. We are fallible, and it always helps to discuss code.